File: 1. C/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    int width = height * 2 + 2;
    int j;

    for (int i = 0; i < height; ++i)
    {
        j = 0;
        while (j < width - height + i + 1)
        {
            ((j >= height - i - 1) && (j < height)) || ((j >= height + 2) && (j <= width - height + i)) ? printf("#") : printf(" ");
            j++;
        }
        printf("\n");
    }
}

File: 6. Python/sentimental-mario-more/mario.py
while True:
    try:
        height = int(input("Height: "))
        if (height in range(1, 9)):
            break
        continue
    except:
        pass

width = height * 2 + 2
for i in range(height):
    j = 0
    while j < width - height + i + 1:
        if ((j >= height - i - 1) and (j < height)) or ((j >= height + 2) and (j <= width - height + i)):
            print("#", end="")
        else:
            print(" ", end="")

        j += 1
    print()
